Driver/Driver/Scenes/LoginScreen/ViewController/LoginViewController.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  LoginViewController.swift
+//  Driver
+//
+//  Created by Fatih Doğan on 22.02.2023.
+//
+
+import UIKit.UIViewController
+
+final class LoginViewController: BaseViewController {
+    
+    // MARK: Outlets
+    @IBOutlet private weak var userNameTextField: UITextField!
+    @IBOutlet private weak var passwordTextField: UITextField!
+    
+    // MARK: Properties
+    var viewModel: LoginViewModelProtocol!
+    
+    // MARK: LifeCycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        userNameTextField.delegate = self
```

Comment:
both setting self should be in viewDidLoad(). We shouldn't set them each this view controller appears.